pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerImpl.java

```diff
@@ -2279,14 +2287,25 @@ private void seekAsyncInternal(long requestId, ByteBuf seek, MessageId seekId, S
                 lastDequeuedMessageId = MessageId.earliest;
 
                 clearIncomingMessages();
-                seekFuture.complete(null);
+                CompletableFuture<Void> future = null;
+                synchronized (this) {
+                    if (!hasParentConsumer && cnx() == null) {
```

Comment:
Disable the behavior to fix `testSeekCustomTopicMessageId`.

Things become complicated when taking care of seeking an internal consumer of a multi-topics consumer.
1. All incoming messages are cleared.
2. Consumer A seeks to a given position
3. All incoming messages are cleared.
4. Consumer B seeks to a given position. **At the same time, consumer A is receiving messages.**
5. All incoming messages are cleared, including the messages received by consumer A.
6. Consumer C seeks to a given position.
7. Call `receive` and the cleared messages of consumer A could never be received.

So the messages of consumer A cannot be received after all these seek operations are done. I think it's an issue with the `MultiTopicsConsumer#seek` so I just get it around in this PR. The race condition should also exist for the previous code. This PR adds more latency to `ConsumerImpl#seek` so the issue is exposed.

Reply:
I opened an issue: https://github.com/apache/pulsar/issues/22208

It's not related to this PR but an existing test could be affected by this PR so I got it around.